src/Adapters/Files/FilesAdapter.js

```diff
@@ -56,6 +59,37 @@ export class FilesAdapter {
    * @return {string} Absolute URL
    */
   getFileLocation(config: Config, filename: string): string {}
+
+  /** Validate a filename for this adaptor type (optional)1G
+   *
+   * @param {string} filename
+   *
+   * @returns {null|*|Parse.Error} null if there are no errors
+   */
+  // TODO: Make this required once enough people have updated their adaptors
+  // validateFilename(filename): ?Parse.Error {}
```

Comment:
It should be fine to uncomment this since external storage adapters won't extend this class.

Adding an entire package for 1 file would never happen.

Reply:
Oops, guess I need to change my other PR.  :-)